release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -275,27 +291,37 @@ In the table below, features are marked with the following statuses:
 |Raw Block with Cinder
 |TP
 |TP
-|
+|TP
 
 |External provisioner for AWS EFS
 |TP
 |TP
-|
+|TP
 
 |CSI volume snapshots
 |TP
 |TP
-|
+|GA
 
 |CSI volume cloning
 |TP
 |GA
 |GA
 
+|vSphere Problem Detector Operator
+|-
+|-
+|GA
```

Comment:
From the following `Graduation Criteria` of vSphere problem detector operator, I prefer it's a TP feature in 4.7.
```
##### Dev Preview -> Tech Preview

* The operator is installed by default in all OCP vSphere clusters and documented in our docs.

##### Tech Preview -> GA

* More testing (upgrade, scale).
* At least 2 releases passed since tech preview to get feedback from customers & support.

```

Reply:
@gnufied - You'd mentioned this is GA. Are you okay with preference from @qinpingli to mark as Tech Preview?

Reply:
Hi @qinpingli - there was considerable discussion about this and the issue is that all features enabled by default are supposed to be GA. This seems contrary to your listed criteria, though, that DP -> TP means operator is installed by default. Eng's preference to list as GA...so I will leave it this way for now.